src/run-custom-command.tsx

```diff
@@ -1,21 +1,30 @@
-import { closeMainWindow, getFrontmostApplication, getPreferenceValues, showHUD } from "@raycast/api";
+import { closeMainWindow, getFrontmostApplication, getPreferenceValues, LocalStorage, showHUD } from "@raycast/api";
 import { doesMatchUrl, getActiveTabUrl, isCurrentAppBrowser, runShortcut } from "./utils";
 import { idToCommandMap } from "./mock";
-import { ShortcutToRun } from "./types";
+import { CommandRecord, ShortcutToRun } from "./types";
 
 export default async function main({ arguments: { id } }: { arguments: { id: string } }) {
   const frontmostApplication = await getFrontmostApplication();
   const { name: frontmostApplicationName } = frontmostApplication;
   const { browser } = getPreferenceValues();
 
-  // todo: look in cache or in local storage by id;
   console.log({ id });
-  const applicationCommandRecord = idToCommandMap[id];
+
+  const allCommandsRaw = await LocalStorage.allItems();
```

Comment:
maybe can be replaced to `getItem`? 
- First check if it is pre-defined command
- if nothing found, `getItem` should be called